modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -3,35 +3,31 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can create a monitor by filling in fields if you do not know how to write tests in the Behavior Driven Development (BDD) test-writing language or how to use the BAT CLI. This method of creating a monitor creates a test that tests one or more endpoints, and schedules the test suite to run at intervals that you specify.
-
-After every interval, you can view the results of the test and statistics about API performance.
+You create monitors via the Functional Monitoring UI or by uploading test suites written in BAT, a Behavior Driven Development (BDD) language. Both methods enable you to create tests against one or more endpoints, and to create test schedules that run at specified intervals. 
 
+This procedure describes creating monitors via the Functional Monitoring UI.
 
 == Before You Begin
 
-* Ensure that your user ID has the Anypoint Monitoring User permission and is assigned the Exchange Administrators role.
-+
-An administrator for your organization can grant you this permission in Anypoint Access Management.
-* Determine which endpoints you want to test and the REST methods that you want to test them with.
-* Decide upon the return codes that you expect the methods to return.
-* Figure out the frequency at which you would like the monitor to test the endpoints.
+* Ensure that your user ID has the Anypoint Monitoring User permission and that it has been assigned the Exchange Administrators role. An administrator for your organization can grant this permission in Anypoint Access Management. For details, see xref:access-management::roles.adoc[Roles].
+* Determine which endpoints to test and the REST methods that test them.
+* Determine the expected return status codes.
+* Determine the test interval.
 +
-The shortest interval is every fifteen minutes. You can also choose to run the monitor hourly, daily, weekly, or monthly. The schedule begins immediately after you create the monitor. You can schedule another monitor later for the same test suite that is more detailed, if you prefer. You can also modify the schedule after you create the monitor, if you want to specify any of these options:
+The shortest interval is fifteen minutes. Monitors can also run hourly, daily, weekly, or monthly. The schedule begins immediately after creating the monitor, and you can modify the schedule after the monitor has been created. For example:
```

Comment:
In general, we spell out single-digit amounts but use digits in mixed single-digit and multidigit contexts.
We use numerals for all _values_ (as opposed to _amounts_: a subtle distinction sometimes).